configuration/api.go

```diff
@@ -50,7 +50,7 @@ func (a *API) UnregisterWebhook(id string) error {
 }
 
 // CreateBot allows to create bot and returns its ID
-func (a *API) CreateBot(name, avatar string, status BotStatus, maxChats uint, defaultPriority GroupPriority, groups []*BotGroupConfig, webhooks *BotWebhooks) (string, error) {
+func (a *API) CreateBot(name, avatar string, status BotStatus, maxChats uint, defaultPriority GroupPriority, groups []*GroupConfig, webhooks *BotWebhooks) (string, error) {
```

Comment:
Can you please split all these functions to separate files, grouped as in docs? (ie. `agents.go`, `bots.go`, `groups.go`, `properties.go` and `webhooks.go`)